### PR TITLE
initial check_in1 markdown

### PR DESCRIPTION
**Pull request by saylerb**
pull request for check_in1 for Robby and me. @robbiejaeger

### CODE REVIEW

File: 1603/module_4_assignments/gametime/template/check_in1.markdown [hunk 1]
```diff
@@ -5,52 +5,108 @@ Complete this form and the README.md before the first check in. Delete this line
...
-e.g.
+- Main Menu
+  - create new game (or join existing game if sockets)
+  - player can select timed game with rounds, or first to 10 points
+
+- Movement
+  - player can move around the screen using arrow keys (or WASD)  
+  - player can move diagonally (press multiple keys)
+  - player movement has realistic physics, arrrow keys accelerate 
+  - ball-to-ball colisions are realistic
```
Comment (robbiejaeger) [inline 1603/module_4_assignments/gametime/template/check_in1.markdown]: collisions

File: 1603/module_4_assignments/gametime/template/check_in1.markdown [hunk 2]
```diff
@@ -5,52 +5,108 @@ Complete this form and the README.md before the first check in. Delete this line
...
+
+- Items and Obstacles
+  - items: player can pickup powerups to increase speed and invulnerability
+  - obstacles: spikes (instant death), barriers
+
+- Multiplayer
+  - node server allows people to join an existing game
```
Comment (robbiejaeger) [inline 1603/module_4_assignments/gametime/template/check_in1.markdown]: node server + sprockets

File: 1603/module_4_assignments/gametime/template/check_in1.markdown [hunk 3]
```diff
@@ -5,52 +5,108 @@ Complete this form and the README.md before the first check in. Delete this line
...
-- Pellets are instructor's faces and they make a sound when they get eaten
-- Cat mode turns the snake into Nyan Cat
+- Scoring
+  - player captures flag in enemy base and returns flag to home base to score points
+  - scoring a point returns the captured flag to its base
+  - rounds are timed to 5 minutes, or first to 10 points
+  - a scoreboard at bottom of the screen indicates current score
+
+- Tagging
+  - when player has the flag, opposing players can tag that player,
```
Comment (robbiejaeger) [inline 1603/module_4_assignments/gametime/template/check_in1.markdown]: no comma

File: 1603/module_4_assignments/gametime/template/check_in1.markdown [hunk 4]
```diff
@@ -5,52 +5,108 @@ Complete this form and the README.md before the first check in. Delete this line
...
-e.g.
+- Main Menu
+  - create new game (or join existing game if sockets)
+  - player can select timed game with rounds, or first to 10 points
+
+- Movement
+  - player can move around the screen using arrow keys (or WASD)  
+  - player can only move one axis at a time (no diagonal, cannot press mulitple directional keys at once) 
+  - player moves at constant rate, acceration is instant
+  - colisions stop players instantly
```
Comment (robbiejaeger) [inline 1603/module_4_assignments/gametime/template/check_in1.markdown]: collisions

File: 1603/module_4_assignments/gametime/template/check_in1.markdown [hunk 5]
```diff
@@ -5,52 +5,108 @@ Complete this form and the README.md before the first check in. Delete this line
...
+  - 2 players can play using arrow keys and WASD 
+
+- Scoring
+  - player captures flag in enemy base and returns flag to home base to score points
+  - scoring a point returns the captured flag to its base
+  - game is first to ~10 points
+  - a scoreboard at bottom of the screen indicates current score
+
+- Tagging
+  - when player has the flag, opposing players can tag that player,
```
Comment (robbiejaeger) [inline 1603/module_4_assignments/gametime/template/check_in1.markdown]: no comma

File: 1603/module_4_assignments/gametime/template/check_in1.markdown [hunk 6]
```diff
@@ -5,52 +5,108 @@ Complete this form and the README.md before the first check in. Delete this line
...
+  - tagging player returns the flag to its base
+  - player doesn't die, just loses flag
+
+- Levels
+  - there is a single maps to choose from
```
Comment (robbiejaeger) [inline 1603/module_4_assignments/gametime/template/check_in1.markdown]: map (singular)

**Comment by rrgayhart** [comment]
@saylerb - You overwrote our actual template - closing until fixed!

**Pull request closed without merging**